data_spec_validator/decorator/decorators.py

```diff
@@ -105,7 +105,7 @@ def _collect_data(method, req_qp, req_data) -> Dict:
 
     def _get_dj_payload(request):
         content_type = request.headers.get('Content-Type')
-        if content_type == 'application/json':
+        if 'application/json' in content_type:
```

Comment:
The previous implementation was not optimal. I suggest we switch to use https://docs.djangoproject.com/en/5.2/ref/request-response/#django.http.HttpRequest.content_type

Reply:
fixed, and also fixed the test utility as using the content_type attribute actually breaks the test cases